libs/langchain-openai/src/chat_models.ts

```diff
@@ -553,6 +555,14 @@ export class ChatOpenAI<
       tool_choice: options?.tool_choice,
       response_format: options?.response_format,
       seed: options?.seed,
+      // Only set stream_options if the model is OpenAI
```

Comment:
Although the python implementation initially defaulted to include usage during streaming, I [updated this](https://github.com/langchain-ai/langchain/pull/22242) to no longer override OpenAI's default due to concerns about no longer having `"finish_reason"` in the final chunk of a stream.